ppr-ui/src/utils/auth-helper.ts

```diff
@@ -41,65 +41,68 @@ export function getKeycloakRoles (): Array<string> {
   throw new Error('Error getting Keycloak roles')
 }
 
+export async function getStaffegisteringParty (isBcOnline: boolean): Promise<PartyIF> {
+  let partyCode = '99990008'
```

Comment:
Can you add these codes as config values? That way if they're ever changed in dev/test/prod we don't need to rerelease. Once you add them into the code I can update 1pass to include them in the deploy

Reply:
done, let me know if it makes sense

Reply:
yeah looks good just need to update how you get the session storage value which I mentioned below